codeworld-server/codeworld-server.cabal

```diff
@@ -50,6 +50,7 @@ Executable codeworld-server
     SafeSemaphore,
     snap-core,
     snap-server,
+    sort,
```

Comment:
You don't need this.  Just use `sort` from `Data.List` in `base`.

Reply:
Ok, thanks.